--- certs.py	2020-02-18 11:14:35.072463989 -0500
+++ certs_new.py	2020-02-20 10:30:48.110464078 -0500
@@ -11,8 +11,11 @@
 If you are packaging Requests, e.g., for a Linux distribution or a managed
 environment, you can change the definition of where() to return a separately
 packaged CA bundle.
-"""
-from certifi import where
+"""	
+def where():
+    """Return the absolute path to the system CA bundle."""
+    return '/etc/pki/tls/certs/ca-bundle.crt'
+
 
 if __name__ == '__main__':
     print(where())
